routes/assetsRoute.js

```diff
@@ -1,37 +0,0 @@
-const path = require('path');
```

Comment:
The fact this commit is a file deletion, and the commit message says its "add"ing something confused me slightly...